### DISCUSSION TITLE
Ultimaker Cura 4.11.0

### DISCUSSION THREAD

**Discussion by Ghostkeeper** [Ultimaker/Cura#10404] [Releases]
_For an overview of the new features in Cura 4.11, please watch [our video](https://youtu.be/OcFkcDyO5DU)._

**Monotonic ordering**
The new Monotonic top/bottom order setting enables users to print parts with smoother top surfaces. This is especially useful for parts that need good aesthetics, such as visual prototypes. Or for parts that benefit from smooth surfaces, such as those that contact-sensitive components.

**Complete UI refresh**
Look around and you will notice that we have refreshed over 100 icons throughout Ultimaker Cura. The new icons are designed for clarity – resulting in a simpler and more informative slicing experience. Also, when scaling the Ultimaker Cura window, the UI will adapt, resulting in less visual clutter.

**Improved digital library integration**
Collaborative workflows using the Digital Library are now simpler. Every user with a cloud-connected Ultimaker 3D printer can access stored projects. And we have added a “Search” function to make finding files easier.

**Save materials profiles to USB**
Users can now save all third-party material profiles to USB. _This feature is for Ultimaker S-line printers only and is especially useful for cloud-connected (or offline) printers._

**Notifications for beta and plugin releases**
Users can now set notification preferences to alert them to new Ultimaker Cura beta and plug-in releases.

**Improve logging of errors in OAuth flow**
When helping a user with log-in problems it is easier to see where the OAuth flow goes wrong.

**Search in the description in the settings visibility menu**
When searching in the settings visibility menu you will also search in the description of the settings.

**Bug fixes:**
- Fixed the setting visibility button to make it easier to click
- Inform the user that their webcam does not work because they are cloud connected
- Inform the user that their webcam does not work if the firewall is enabled
- Fixed a crash when pressing the slice button while context menu is opened 
- Support non-ASCII character in the Digital Library project name
- Fixed integer underflow if print is less than half the initial layer height
- Fixed a bug where infill mesh sometimes default to having walls or skin
- Fix builds with Python 3.8, contributed by StefanBruens
- Fix CC settings for PLA
- Fixed memory leak in Zeroconf 0.25
- Fixed connecting USB printing with detecting baud-rates, contributed by rrrlasse
- Fixed crash when Cura crashes on exit
- Fixed a bug where the infill goes through walls
- Fixed the version upgrade of preferences file
- Fixed missing icons in deprecated icons list, contributed by fieldOfView
- Fixed a crash in CuraEngine when the prime tower is placed in an invalid position
- Fixed a bug when user is unable to sign in on Linux if a Keyring backend is installed
- Fixed the rotation direction of the 90 degrees rotation arrows, contributed by fieldOfView

**Printer definitions, profiles and materials:**
- Added SecKit SK-Tank, SK-Go printer definitions, contributed by SecKit
- Added MP Mini Delta 2 printer definition, contributed by PurpleHullPeas
- Added Kingroon K3P and K3PS printer definitions, contributed by NoTaMu
- Added Eryone PLA, PLA Wood, PLA Matte and PETG 1.75mm profiles, contributed by dapostol73
- Added BIQU BX printer definition, contributed by looxonline
- Added FLSun Super race printer definitions, contributed by thushan
- Added Atom 2.0 and Atom Plus printer definitions, contributed by lin-ycv
- Added PBR 3D Gen-I printer definition, contributed by pbr-research
- Added Creasee 3D printer definitions, contributed by ivovk9
- Updated Strateo3D profiles, contributed by ChronosTech
- Added Voron V0 printer definitions, contributed by jgehrig
- Updated Liquid profiles, contributed by alexgrigoras
- Added Farm 2 and Farm2CE printer definitions, contributed by saliery999
- Added GooFoo and Renkforce print definitions and GooFoo materials, contributed by goofoo3d

_From version 4.11 onwards - Ultimaker Cura is only supported on operating systems actively maintained by their software manufacturer or community. This means Windows 7 and MacOS 10.13 will no longer be supported. Technically this means Ultimaker will stop testing and developing for such operating systems. However, even though it is no longer supported, there is still a high likelihood the application keeps functioning._

<hr /><em>This discussion was created from the release <a href='https://github.com/Ultimaker/Cura/releases/tag/4.11.0'>Ultimaker Cura 4.11.0</a>.</em>

**Comment by phil2sat** [comment]
Cura latest Beta does generate gcode with M106 S255 even if every fan setting is set to 0 or completely disabled.

`;TYPE:SKIN
M106 S255
G1 F1500 E0.8`

I guess here comes some monotonic infill.

very bad for me because at this point my printer stops due to thermal runaway, so its acually not usable

**Reply by nallath** [reply]
Reported here: https://github.com/Ultimaker/Cura/issues/10407
Fixed in: https://github.com/Ultimaker/CuraEngine/pull/1500

Please try to look in the closed issues first; It could well be that it was already reported (and fixed).

**Comment by aktivo** [comment]
I can not slice in any version greater than 4.8. When I click on slice button, say: "slicing ...", but nothing happend. I wait for one hour - nothing happened - 0 %. I try it on clear Windows 10 - same problem. I try it with Creality Ender 3 Pro printer.

**Reply by nallath** [reply]
Please make a bug report out of this issue. We need more information than this to solve it.

**Reply by phil2sat** [reply]
A shot in the blue, your firewall is blocking the slicer connection... but without further information or a log file im just guessing

**Reply by aktivo** [reply]
Here is log:
[cura.log](https://github.com/Ultimaker/Cura/files/7213300/cura.log)

**Reply by aktivo** [reply]
I allow it on firewall and now works fine. Thanks for tip.

**Comment by IngwarSa** [comment]
Hello. I have a problem when printing curves. The inner walls are printed in spurts. For example, I made a cylinder model, studied the CURA G-code and SIMPLIFY3D. I took code snippets for the interior and exterior walls. Based on this data, I built diagrams.
This is the SIMPLIFY3D code
![Simply](https://user-images.githubusercontent.com/92488422/137221346-73fa2e3e-dcb7-4952-954b-797bd486909e.png)
This is the CURA code
![Cura](https://user-images.githubusercontent.com/92488422/137221395-26106b38-c7d4-413d-ad54-5ef78f650b0c.png)

I have tried printing thin walls without them. The result has not changed.
The part has a wall of 1.2 mm. The nozzle diameter is 0.4 mm. The speeds for the inner and outer walls are different.
How do I remove uneven movement when printing curves of the inner walls?

**Reply by Ghostkeeper** [reply]
I suggest setting the Minimum Wall Flow higher (resulting in slight underextrusion if the part is not exactly 1.2mm everywhere) or disabling the Compensate Inner Wall Overlaps setting (resulting in significant overextrusion).

I normally keep my Minimum Wall Flow around 50%.

I think you're running into https://github.com/Ultimaker/Cura/issues/6748.

**Reply by IngwarSa** [reply]
There are more images of the problem here.
(https://3dtoday.ru/questions/dergane-pri-proxode-krivyx-vnutrennix-stenok-na-cura#comment-823518)

**Reply by Ghostkeeper** [reply]
Those are the same images. Did you try disabling Compensate Inner Wall Overlaps?

**Reply by IngwarSa** [reply]
Yes, I tried it. Does not help. On version 4.12, the same.
![1](https://user-images.githubusercontent.com/92488422/139288222-81696ece-44d0-457d-bee6-1f71ec24d98f.png)

**Comment by Ghostkeeper** [comment]
Ah, yes I see. It's gap filling then. Cura 5.0 (Arachne project) is intended to fix this issue, but it's been a problem for a long time: https://github.com/Ultimaker/Cura/issues/5110

**Comment by Toddi3D** [comment]
hello, after install Cura 4.12.1 the Cura Symbols for .stl Datas are gone, there is nothing. Anybody can help ?

best Regards and say healthy,

**Reply by Ghostkeeper** [reply]
Might want to look into what the file association of STL files is in Windows, then. Here's a guide: https://fileinfo.com/help/windows_10_change_file_associations
Or you could run the installer again and make sure to look into what it says in the file association when it passes by in the wizard?

**Comment by CHSIVASANTHOSH** [comment]
I am unable to build cura form source can anyone suggest me the best way to build , the steps mentioned are not working fine for me

**Reply by jellespijker** [reply]
Cura from source works only if you follow the instructions provided in the Wiki https://github.com/Ultimaker/Cura/wiki/Running-Cura-from-Source .
Keep in mind that building older version like this one 4.11.0 is hard and we can't support you with that.

If you still have problems with that process or Cura 5.3..0-alpha (current main branch) please open a regular issue instead of using these discussions.
